composer.json

```diff
@@ -6,16 +6,24 @@
         {
             "name": "Caleb Porzio",
             "email": "calebporzio@gmail.com"
+        },
+        {
+            "name": "Tom Witkowski",
+            "email": "dev.gummibeer@gmail.com",
+            "homepage": "https://gummibeer.de",
+            "role": "Developer"
         }
     ],
     "require": {
-        "illuminate/database": "~5.8.0|^6.0.0",
-        "illuminate/support": "~5.8.0|^6.0.0",
-        "illuminate/filesystem": "~5.8.0|^6.0.0"
+        "php": "^7.4",
```

Comment:
No ! This could not even work with Laravel 5.8 and PHP 7.1.3
This would make - check https://github.com/laravel/laravel/blob/5.8/composer.json#L11

Reply:
It's because of some features I've used `??=`, typed properties and so on. So far I know there's no conflict between Laravel5.8 and PHP7.4?

Reply:
What when you run Laravel5.8 with PHP 7.1.3 ?

Reply:
You can't install it, which is the base of composer and it's not the first package that requires a higher PHP version than the lowest Laravel version it supports.
So if there's not a conflict that makes running Laravel 5.8 on PHP7.4 impossible there's no problem with this combination except the fact that users with a php version below 7.4 can't use this.

Reply:
Which, we should note, is now the majority of people. Not everyone has already, or even can upgrade their php version to 7.4

Reply:
We can close this topic. Caleb said that he will stick to PHP7.1 and there's nothing that wouldn't be possible with php7.1.

So how about returning to the real question: which, if any, features would be good to have in this package.